client/scripts/views/modals/onboarding_modal/choose_address.ts

```diff
@@ -0,0 +1,454 @@
+import 'modals/onboarding_modal/choose_address.scss';
+
+import m, { Vnode } from 'mithril';
+import $ from 'jquery';
+
+import { stringToHex } from '@polkadot/util';
+import { InjectedAccountWithMeta } from '@polkadot/extension-inject/types';
+import { SignerPayloadRaw } from '@polkadot/types/types/extrinsic';
+
+import { SigningCosmosClient } from '@cosmjs/launchpad';
+
+import mixpanel from 'mixpanel-browser';
+import app, { ApiStatus } from 'state';
+import { initChain, initAppState, selectNode } from 'app';
+import { isSameAccount, link } from 'helpers';
+import { Button, Input, TextArea, Spinner, Checkbox } from 'construct-ui';
+import { notifyError, notifyInfo } from 'controllers/app/notifications';
+import { AddressInfo, Account, ChainBase, IWebWallet } from 'models';
+import { confirmationModalWithText } from 'views/modals/confirm_modal';
+import { baseToNetwork, networkToBase } from 'models/types';
+
+import { validationTokenToSignDoc } from 'adapters/chain/cosmos/keys';
+import { updateActiveAddresses, createUserWithAddress, setActiveAccount } from 'controllers/app/login';
+import Substrate from 'controllers/chain/substrate/main';
+import Ethereum from 'controllers/chain/ethereum/main';
+import Near from 'controllers/chain/near/main';
+import { SubstrateAccount } from 'controllers/chain/substrate/account';
+import EthereumAccount from 'controllers/chain/ethereum/account';
+
+import CodeBlock from 'views/components/widgets/code_block';
+import User from 'views/components/widgets/user';
+import AvatarUpload from 'views/components/avatar_upload';
+
+import { onboardingChooseAddressIcon } from '../../components/sidebar/icons';
+import { ChainBaseIcon } from '../../components/chain_icon';
+import OnboardingFooterActions from './footer_actions';
+import AddressSwapper from '../../components/addresses/address_swapper';
+
+const EthereumLinkAccountItem: m.Component<{
```

Comment:
These link account items all got refactored into a single component type on master -- should ensure that change persists, as well as other changes to the `link_new_address_modal` that have occurred since this file was created.

Reply:
fixed